fixtures/packaging/webpack/config.js

```diff
@@ -1,9 +1,11 @@
+var path = require('path');
+
 module.exports = {
   entry: './input',
   output: {
     filename: 'output.js',
   },
   resolve: {
-    root: '../../build/packages',
+    root: path.resolve('../../../build/packages/'),
```

Comment:
@gaearon looks like [webpack requires](https://webpack.github.io/docs/configuration.html#resolve-root) `resolve.root` to be absolute, had to make this change for the webpack packaging fixtures to build